accounts/admin.py

```diff
@@ -1,3 +1,35 @@
 from django.contrib import admin  # noqa
 
 # Register your models here.
+from accounts.models import User
+
+
+class UserAdmin(admin.ModelAdmin):
+    list_display = [
```

Comment:
Are all the fields? If they are all you can use `list_display = __all__` (I think is like that, check the documentation).
If they are not all, I'm ok with your changes.
Thanks ;)

Reply:
I'm not sure if there are all fields, anyway, I don't recommended to show all fields in list. Only the essentials.

For example:

* email
* first_name
* last_name
* is_active

And not more.

Reply:
Ok, I'll only add date_joined to enable one date filter.